CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Tabstops not being reset with `reset`
 - Selection not cleared when switching between main and alt grid
 - Fallback to `LC_CTYPE=UTF-8` on macOS without valid system locale
+- Emoji colors do not blend with background and are displayed in their true colors
```

Comment:
```suggestion
- Emoji colors blending with terminal background
```

We should state what was fixed, the way it's currently written it sounds a bit like it was fixed that they do not blend, but we fixed that it was blending incorrectly.

Reply:
Applied